target/linux/lantiq/config-4.4

```diff
@@ -178,7 +179,6 @@ CONFIG_RTL8366_SMI=y
 CONFIG_SCHED_HRTICK=y
 # CONFIG_SCHED_INFO is not set
 # CONFIG_SCSI_DMA is not set
-# CONFIG_SENSORS_LTQ_CPUTEMP is not set
```

Comment:
Ok agree, but this should not happend! If i do `make kernel_menuconfig CONFIG_TARGET=target` then the system should prepare the whole tree for lantiq independently of the subtarget. The command should set the missing config param in target if it is not set in the subtarget.
How do you do this kind of changes? If it is working as it is then it is error prone.